### ISSUE TITLE
Eliminate generation of empty "accesskey" attributes: 1

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#272]
Some template(s) are producing an empty "accesskey" attribute on secondary menu item ```<li>``` elements. It should be eliminated unless it's supposed to have a value in it, in which case we should actually populate the value. 

> The attribute accesskey is being used for the secondary menu items with no value. If there are attributes there should also be a value or the attribute should be removed.

![screen shot 2018-04-27 at 3 50 21 pm](https://user-images.githubusercontent.com/263285/39384525-b917d796-4a32-11e8-8ae5-518f6bab991e.png)

WCAG 2.0 Compliance: [4.1.1. Parsing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

> In content implemented using markup languages, elements have complete start and tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features. (Level A)

> Resource: https://www.w3.org/TR/WCAG20-TECHS/H44.html

Traceability: [Microassist Report 1 > WCAG Compliance > Parsing](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.9b291xk40aeb)

**Comment by paregorios** [comment]
This is working on production.
